util/color.lua

```diff
@@ -14,6 +14,48 @@ local colorTable = {
   [489] = {1, 0.5, 0}
 }
 
+-- Color criteria based on the item level relative to the player's average item level
+local colorCriteria = {
+  orange = {1, 0.5, 0},      -- Above or equal to average item level
+  purple = {0.63, 0.21, 0.93}, -- 5 or less lower than average item level
+  blue = {0, 0.55, 0.87},    -- 6-10 lower than average item level
+  green = {0, 1, 0},         -- 10-20 lower than average item level
+  yellow = {1, 1, 0}         -- More than 20 lower than average item level
```

Comment:
better to use grey. Yellow is already associated with quest content, and grey is associated with low-quality items.